### DISCUSSION TITLE
Observations

### DISCUSSION THREAD

**Discussion by dustymc** [ArctosDB/arctos#5250] [General]
From AWG meeting (https://docs.google.com/document/d/1VEUSR-8UK0-9WeFOyiJDRit9UCIDJbq-HMpXUbBRvm8/edit#) involving discussion of https://github.com/ArctosDB/arctos/issues/1302

It's clear that we don't have a best practices to handling observations, and we need one. Here's a start.

# Things that are completely incompatible with Arctos

Using parts as representations of things to which barcodes cannot be stuck. It's just indefensible, completely incompatible with the model. (And maybe that's not desirable, but changing it without making a giant mess of everything would require a different model, not just different terminology.) 

# Worst Practices

Things that to me look like they won't quite get the idea across while making future-you very sad. 

* Undoing the disposition cleanup by introducing a bunch of creative ways to say "we don't have it." (What precisely disposition *is* still needs resolved, I maintain that it is a lightweight detail-free way to begin answering 'can I have it?' and little/nothing more.)
* Doing some elaborate dance (potentially involving disposition and various part attributes) which attempts to indicate that the part is not in fact something to which one might stick barcodes
* Trying to infer the purpose of the record through event type
* Trying to say something about parts through event type
* Trying to infer anything at all from collection type ("normal" collections catalog observations, "Obs" collections catalog skin clips, there's just no connection)

# Best Practices

## Flagging records as observations

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctcataloged_item_type#observation

## Loaning digital media

Archive collections!

# Missing

## Depicted in media

We seem to have a desire to say things like "this record is based off of a [ digital picture | written description | vague memory ] of a skull." I think that's media-things (https://arctos.database.museum/info/ctDocumentation.cfm?table=ctmedia_label#subject - ish), not anything about catalog record. (But maybe there's some wraparound via archive collections?)

I think I've recommended using https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecpart_attribute_type#representation for this, but occasionally/mostly/currently I think it should be reserved for physical things (painting of a skull, 3D print of a CT of a skull), not eg pixels in a digital image.

These conversations often seem to involve NOT being asked to borrow this stuff, maybe that's evidence that something like https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#description (eg human-readable description of why the record exists rather than anything that might be categorized) would be a better approach.

## Search

Finding catalog records by attributes of media attached to them, whatever form that might take, probably isn't quite impossible in SQL, but it's also the kind of thing lucene/elastisearch/solr/etc. exist to accommodate. Proposal time?

@Jegelewicz @ccicero @Nicole-Ridgwell-NMMNHS @ebraker @wellerjes @mkoo @genevieve-anderegg @campmlc @atrox10 and anyone else who cares about observations, please chime in.

**Comment by Jegelewicz** [comment]
Want to meet up and talk it out? Provide your availability [here](https://www.when2meet.com/?17564589-O885i). I will pick a day/time on November 8.

**Comment by Jegelewicz** [comment]
Meeting set for November 28 at 2PM MST

Also saw these things for managing camera trap data in the GBIF webinar this am
https://www.agouti.eu/
https://www.wildlifeinsights.org/
